### ISSUE TITLE
[pkg.jenkins.io,releases] Finish cleanup of mirrorbrain

### ISSUE THREAD

**Issue by dduportal** [jenkins-infra/helpdesk#2970]
I thought that #2888 was finished, but https://github.com/jenkins-infra/helpdesk/issues/2967 clearly demonstrates that there are still some leftovers of mirrorbrains:

- User `mirrorbrain` still present on the machine and used by the releases
  - https://github.com/jenkins-infra/release/blob/1979e858fbaaacaaf54ed10122d11dbc0ba79d93/utils/release.bash#L610
  - https://github.com/jenkins-infra/release/blob/6669feed8e31482be18cb1b3516bdd87ec4f6d06/profile.d/experimental#L19
  - Fix the runbook https://github.com/jenkins-infra/runbooks/blob/e36ae6588fd03b3a2220f620a7db4fd1f49efe69/security/README.adoc#L55
  - Manage blobxfer installation (and any dependency of the mirror scripts)
- Put back into Puppet management the [mirror-scripts](https://github.com/jenkins-infra/mirror-scripts/)
- Correct comment at https://github.com/jenkins-infra/jenkins-infra/blob/65308dd71c7f17bbb1c96608647bfb97bdcdff64/dist/role/manifests/pkg.pp#L2
- Fix the public documentation: https://github.com/jenkins-infra/jenkins.io/blob/002ad0b3450822ffe7b68456927f678b4125db91/content/projects/infrastructure/index.adoc#L111

Closes #505

**Comment by dduportal** [comment]
We have https://github.com/jenkins-infra/helpdesk/issues/505 that we want to revert as part of this cleanup

**Comment by dduportal** [comment]
Stuff to NOT forget to migrat from the former mirrobrain.pp profile + from the production VM (and that has been managed manually)

- https://github.com/jenkins-infra/helpdesk/issues/3034#issuecomment-1177869488 
- https://github.com/jenkins-infra/helpdesk/issues/3034#issuecomment-1177890937

**Comment by dduportal** [comment]
While working on https://github.com/jenkins-infra/helpdesk/issues/3414 we realized the following:

- The user `mirrorbrain` is too much coupled with the whole `pkg` VM (pkg.origin.jenkins.io and updates.jenkins.io). It was used for multiple purposes: the former "mirrors.jenkins.io" (mirrorbrain) service, but also by all the mirrors rsyncs as per #505 and https://github.com/jenkins-infra/helpdesk/issues/3414#issuecomment-2025472147


As such, we decided to bring back the `mirrorbrain` user under puppet management in https://github.com/jenkins-infra/jenkins-infra/pull/3357 as we keep using it.
Note that it implied only a minimalist set of resources: the user itself, only one of its crontabs (used for full sync), current SSH keys for archives.j and OSUOSL servers.

Last mile before closing this issue: https://github.com/jenkins-infra/jenkins-infra/issues/3352
